### ISSUE TITLE
SD card controller should provide F011 emulation registers at $D080

### ISSUE THREAD

**Issue by gardners** [gardners/c65gs#28]
The SD card controller should have the facility to pretend to be the F011 FDC controller.  This will require mapping logical 1581 sector numbers to physical SD card sectors.
On the upside, the F011 has an 8-bit register for side, which removes one barrier to supporting DHD images in future.

Depends on Issue #24.

**Comment by gardners** [comment]
Closing this issue, and will create a new separate issue for the F011 write problems.
